src/main.rs

```diff
@@ -72,8 +65,18 @@ fn run() -> Result<(), MainError> {
                     .pipe(Err)
             }
         }
-        "install" | "i" | "update" | "up" => pass_to_pnpm(&cli.name[0..]),
-        _ => pass_to_sub(cli.name[0..].join(" ")),
+        Some(cli::Command::Install(passed_trough_args)) => {
+            let mut args = passed_trough_args.args;
+            args.insert(0, "install".into());
+            pass_to_pnpm(&args)
+        }
+        Some(cli::Command::Update(passed_trough_args)) => {
+            let mut args = passed_trough_args.args;
+            args.insert(0, "update".into());
+            pass_to_pnpm(&args)
+        }
+        Some(cli::Command::Other(args)) => pass_to_sub((&*args.join(" ")).into()),
+        None => panic!("Couldn't parse the CLI args"),
```

Comment:
So my choice of `Option<Command>` as the type of `Cli::command` forces you to do this awkward panic. You can fix this by change the type of field `command` of the `Cli` struct to just `Command`. I initially wrote `Option<Command>` because I thought you would want to handle the case where there is no command.